media/js/mozorg/home/home.js

```diff
@@ -66,4 +70,68 @@
 
     // Video card interactions.
     initVideoEvents();
-})();
+
+    /*
+     * Sticky CTA
+     */
+
+
+    var $stickyCTA = $(document.getElementById('download-firefox-sticky-cta'));
+    var hasCookies = typeof Mozilla.Cookies !== 'undefined' && Mozilla.Cookies.enabled();
+    $stickyCTA.attr('aria-hidden', 'true');
+
+    // init dismiss button
+    function initStickyCTA() {
+        // add and remove aria-hidden
+        var primaryTop = new Waypoint({
+            element: document.getElementById('download-firefox-primary-cta'),
+            handler: function(direction) {
+                if(direction === 'down') {
+                    // becomes percivable as the user scrolls down
+                    $stickyCTA.removeAttr('aria-hidden');
+                } else {
+                    // hidden again as they scroll up
+                    $stickyCTA.attr('aria-hidden', 'true');
+                }
+            }
+        });
+
+        // add button
+        var $dismissButton = $('<button>').addClass('sticky-dismiss').text('Dismiss download prompt.');
+        var $stickyWrapper = $stickyCTA.find('.primary-wrapper');
+        $dismissButton.appendTo($stickyWrapper);
+        // listen for click
+        $dismissButton.one('click', function(){
+            // dismiss
+            dismissStickyCTA(primaryTop);
+        });
+    }
+
+    // handle dismiss
+    function dismissStickyCTA(wayPoint) {
+        // destroy waypoint
+        wayPoint.destroy();
+        // remove element
+        $stickyCTA.remove();
+        // set cookie, if cookies are supported
+        if (hasCookies) {
+            var d = new Date();
+            d.setTime(d.getTime() + (30 * 24 * 60 * 60 * 1000)); // 30 days
+            Mozilla.Cookies.setItem('sticky-home-cta-dismissed', 'true', d, '/');
+        }
+    }
+
+    // Check if previously dismissed
+    if (hasCookies) {
+        if (!Mozilla.Cookies.getItem('sticky-home-cta-dismissed')) {
+            // init the button
+            initStickyCTA();
+        } else {
+            $stickyCTA.remove();
+        }
+    } else {
+        // they can still dismiss it and get ARIA with cookies disabled
+        initStickyCTA();
+    }
```

Comment:
I'm of the mind that we shouldn't show the banner if dismissing it won't be remembered. I could be persuaded otherwise though.

(Maybe just don't show the "x" if `hasCookies === false`? Though that may be more work than just never showing the banner to begin with.)

Reply:
I'm fine not showing it.